voting/src/general_meeting.rs

```diff
@@ -35,6 +38,14 @@ impl Context {
     fn storage_mut(&mut self) -> &mut dyn SubStorageAccess {
         self.storage.as_mut().unwrap().as_mut()
     }
+
+    fn admin(&self) -> Public {
+        let bytes = self
+            .storage()
+            .get(ADMIN_STATE_KEY.as_bytes())
+            .expect("GeneralMeeting module set the adin in the genesis state");
```

Comment:
There is a typo here admin.

Reply:
Please use "Request changes" when you review a PR.